### PR TITLE
Update sport list and sub sports

### PR DESCRIPTION
**Pull request by vlcvboyer**

### CODE REVIEW

**Comment by vlcvboyer** [comment]
#4313

**Comment by vlcvboyer** [comment]
#4312

File: src/FileIO/FitRideFile.cpp [hunk 1]
```diff
@@ -3827,12 +3827,17 @@ void write_session(QByteArray *array, const RideFile *ride, QHash<QString,RideMe
 
     // 6. sport
     // Export as bike, run or swim, default sport is bike. 
-    // todo: support all sports based on tag "Sport"
-    int sport = ride->isRun() ? 1 : ride->isSwim() ? 5 : 2;
-    write_int8(array, sport);
+    value=FitFileReaderState::getSportId(ride->getTag("Sport",""));
```
Comment (amtriathlon) [inline src/FileIO/FitRideFile.cpp:4160]: Here would be better to use
https://github.com/GoldenCheetah/GoldenCheetah/blob/68c795440c022edf133bf803691af9b57905d6aa/src/FileIO/RideFile.cpp#L246
which already does the sport tag "normalization" due to translations, etc.

Comment (vlcvboyer) [reply src/FileIO/FitRideFile.cpp]: correct
PR updated accoringly

**Comment by amtriathlon** [comment]
Thank you!

**Pull request merged**